test/integration/cloud-config-aws.yml.template

```diff
@@ -14,6 +14,8 @@
 aws_access_key: @ACCESS_KEY
 aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
+aws_region: @REGION
```

Comment:
Last time I checked, only some of our AWS integration tests support using a region other than us-east-1. That is one of the reasons why ansible-test doesn't support substitution of a region variable. If this is something we're going to support, the tests will need to be fixed as well. Keep in mind that tests will still only run in CI on us-east-1, so there are likely to be regressions in the multi-region support after we fix it.